README.md

```diff
@@ -23,6 +23,8 @@ Because open-source Kubernetes depends on your specific configuration, Pega does
 
 The helm charts currently only support running on Kubernetes nodes with x86-64 CPUs.  ARM CPUs are currently unsupported.
 
+The helm charts support running on Kubernetes versions that are in Maintenance and Active Support at the time of the helm chart’s release. For more information, see [Kubernetes EOL policy](https://endoflife.date/kubernetes).
```

Comment:
Based on that referenced article, I think it would be better to use the Kubernetes preferred terminology releases. As the bug states, it might be useful to add in details about how to get to the helm charts release page>>
The helm charts support running on Kubernetes releases that are in Maintenance and Active Support at the time of the helm chart’s release. To review the data of a Pega helm chart release, see [Releases](https://github.com/pegasystems/pega-helm-charts/releases); to review the latest matrix of Kubernetes release support, see [Kubernetes EOL policy](https://endoflife.date/kubernetes).

Reply:
Great suggestions, thanks!